classes/Product.php

```diff
@@ -52,6 +52,9 @@ class ProductCore extends ObjectModel
     /** @var int default Category id */
     public $id_category_default;
 
+    /** @var array */
```

Comment:
Hello @LittleBigDev 
I've no idea of the context, just an alert, is it really array ?